runtimes/node/versions/latest/src/server-next-js.mjs

```diff
@@ -1,11 +1,22 @@
 import { parse } from "url";
 import next from "next";
 import express from "express";
+import { readFileSync } from "fs";
 import { onInit, getPort, getHost, onAction, onError } from "./ssr/helpers.mjs";
 
 console.log("Next.js server starting ...");
 
 const app = express();
+
+app.use((req, res, next) => {
+  if (req.path === "/__opr/timings") {
+    const timings = readFileSync("/mnt/telemetry/timings.txt", "utf8");
+    res.setHeader("content-type", "text/plain; charset=utf-8");
+    return res.status(200).send(timings);
+  }
+  next();
+});
```

Comment:
Notice just below this, we have app.use(onInit), onInit coming from helpers.mjs.
Let's make this into helper function too, and do app.use(telemetryEndpoint) or such.

This makes it easier in future when we need to update the code, it would be in just 1 place for all ssr frameworks